test/game_teams_collection_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/game_teams_collection'
+require './lib/game_team'
+require 'csv'
+
+class GameTeamsCollectionTest < Minitest::Test
+
+  def test_it_exists
+    gtc = GameTeamsCollection.new('./test/fixtures/game_teams.csv')
+    assert_instance_of GameTeamsCollection, gtc
+  end
+
+  def test_has_a_path
+    gtc = GameTeamsCollection.new('./test/fixtures/game_teams.csv')
+    assert_equal './test/fixtures/game_teams.csv', gtc.path
+  end
+
+  def test_it_can_read
```

Comment:
We'll probably want to tighten up our testing here, and make sure the other files have similar testing and testing for edge cases, and proper reading/creation of the instances.

Looks good though!